.github/workflows/test-package-build.yml

```diff
@@ -0,0 +1,68 @@
+name: Test package build
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    types:
+      - opened
+      - synchronize
+      - reopened
+      - ready_for_review
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+        matrix:
+            python-version: ['3.9', '3.10']
+    timeout-minutes: 20
+    steps:
+        - name: Checkout code
+          uses: actions/checkout@v4
+        - name: Set up Python
+          uses: actions/setup-python@v5
+          with:
+            python-version: ${{ matrix.python-version }}
+        - name: Install dependencies
+          run: |
+            python -m pip install --upgrade pip
+            pip install hatch
+            pip install -r requirements/test-requirements.txt
+        - name: Build distribution files
+          id: build-dist
+          run: |
+            hatch build
+            # List distribution files and check their file sizes
+            ls -lh dist
+
+            # Set step outputs
+            sdist_path=$(find dist -type f -name "*.tar.gz")
+            wheel_path=$(find dist -type f -name "*.whl")
+            wheel_name=$(basename $wheel_path)
+            wheel_size=$(stat -c %s $wheel_path)
+            echo "sdist-path=${sdist_path}" >> $GITHUB_OUTPUT
+            echo "wheel-path=${wheel_path}" >> $GITHUB_OUTPUT
+            echo "wheel-name=${wheel_name}" >> $GITHUB_OUTPUT
+            echo "wheel-size=${wheel_size}" >> $GITHUB_OUTPUT
+        - name: List files in source distribution
+          run: |
+            tar -tf ${{ steps.build-dist.outputs.sdist-path }}
+        - name: List files in binary distribution
+          run: |
+            unzip -l ${{ steps.build-dist.outputs.wheel-path }}
```

Comment:
We can remove these steps. MLflow includes lots of non-python files so inspecting them makes sense, but this package doesn't.